tests/components/youless/snapshots/test_sensor.ambr

```diff
@@ -611,6 +713,62 @@
     'state': 'unknown',
   })
 # ---
+# name: test_sensors[sensor.power_meter_tariff-entry]
+  EntityRegistryEntrySnapshot({
+    'aliases': set({
+    }),
+    'area_id': None,
+    'capabilities': dict({
+      'options': list([
+        '1',
+        '2',
+      ]),
+    }),
+    'config_entry_id': <ANY>,
+    'device_class': None,
+    'device_id': <ANY>,
+    'disabled_by': None,
+    'domain': 'sensor',
+    'entity_category': None,
+    'entity_id': 'sensor.power_meter_tariff',
+    'has_entity_name': True,
+    'hidden_by': None,
+    'icon': None,
+    'id': <ANY>,
+    'labels': set({
+    }),
+    'name': None,
+    'options': dict({
+    }),
+    'original_device_class': <SensorDeviceClass.ENUM: 'enum'>,
+    'original_icon': None,
+    'original_name': 'Tariff',
+    'platform': 'youless',
+    'previous_unique_id': None,
+    'supported_features': 0,
+    'translation_key': 'active_tariff',
+    'unique_id': 'youless_localhost_tariff',
+    'unit_of_measurement': None,
+  })
+# ---
+# name: test_sensors[sensor.power_meter_tariff-state]
+  StateSnapshot({
+    'attributes': ReadOnlyDict({
+      'device_class': 'enum',
+      'friendly_name': 'Power meter Tariff',
+      'options': list([
+        '1',
+        '2',
+      ]),
+    }),
+    'context': <ANY>,
+    'entity_id': 'sensor.power_meter_tariff',
+    'last_changed': <ANY>,
+    'last_reported': <ANY>,
+    'last_updated': <ANY>,
+    'state': 'unknown',
```

Comment:
Why is this unknown?

Reply:
I figured it out. the version of the firmware in the test was set to old. That was why the integration never loaded the phase information. Update incoming to fix that issue.

Reply:
But if that is the case, we should prevent creating these entities in the first place if the version is too low. So ideally we add a check for that and we have tests for that